id_rsa.pub

```diff
@@ -1 +0,0 @@
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDm9LVUeTxwZPwXpDk92pwfqIIProXscQcEWXqlF5NVfmwJFW1XhsPC+0FgHnOwN5SReCtlxHS6WxYKdAxu8F6L1zAsL7JfUBJMrN/dRLbS2fInwyL95OTXsmgjniRFk6mzjgDoWG/5X4waUK2ArlX4Pqx6fHaoJrg6sFxf5t6hLOdP6eYidPVEqOC4g0CkRM8YH9NbXq7lUo+kFhzqkQfopP8RLD58nNfi9pbU7b/wuETmmYg/og2XLIY9J6KzLUO5y7B4sf5QDAUV7+Il2Qe35I6ta46w8dHQKPqV8jdMWpeuKHorO3Kya4kn61jjsaODZuXC01CAoCSFHemODtQD chana@01123-COMMS
```

Comment:
If you've removed this and `id_rsa.enc`, you'll have broken auto-publishing.

Reply:
Yarp. 

Me: "What are these doing here _deletes_".

TIL

Reply:
Have replaced.